ionoscloud/firewall_service.go

```diff
@@ -0,0 +1,128 @@
+package ionoscloud
```

Comment:
@cristiGuranIonos the file is named `firewall_service.go` but it isn't stored in `services` directory, this is a little bit confusing. Is there a reason for which we didn't place the file in `services` directory in the first place?

Reply:
import cycle, like the label service. Will try to refactor, but in a different review since it's a big refactor